CITATION

```diff
@@ -0,0 +1,29 @@
+If you use iminuit for work/research presented in a publication, we recommend and encourage to add the following two references:
+
+Piti Ongmongkolkul and Christoph Deil and Hans Dembinski and David Men\'endez Hurtado and Chris Burr and Andrew ZP Smith and Fabian Rost and Alex Pearce and Lukas Geiger and Omar Zapata, iminuit - A Python interface to MINUIT, 2012 --, https://github.com/iminuit/iminuit; accessed online <your current date>.
+
+The original Minuit paper.
+http://adsabs.harvard.edu/abs/1975CoPhC..10..343J
+
+Recommended BibTeX entries for the above citations are:
+
+@ONLINE{iminuit,
+  author = {Piti Ongmongkolkul (@piti118) and Christoph Deil (@cdeil) and Hans Dembinski (@HDembinski) and David Men\'endez Hurtado (@Dapid) and Chris Burr (@chrisburr) and Andrew ZP Smith (@energynumbers) and Fabian Rost (@fabianrost84) and Alex Pearce (@alexpearce) and Lukas Geiger (@lgeiger) and Omar Zapata (@omazapa)},
```

Comment:
Is it a good idea to put the Github usernames in the author field?
Does this even render OK in LaTeX?
I think most people will not understand what this is, so I'm -1 to put it in the bibtex entry.

Adding a contributor list to our docs would be good, and there you could link to the Github handles.
This I find very convenience, I've done this e.g. for another project I work on: http://gammapy.org/team.html#contributors

Reply:
Sounds good, that I did not remove the github handles was an oversight, not intentional. Should be skip the names and just put "iminuit collaboration"? And then list the names in a "contributors" section in the docs? I like this, because keeping this information in sync and up-to-date is a pain.

We want to have only one place where we put the contributors, it should be a "contributors" section or a CONTRIBUTORS file in the repo.

Reply:
Yes, I would prefer "iminuit contributors" here, and only have to maintain a contributor list in one place, somewhere in the docs.